internal/cloud/realm/dependencies.go

```diff
@@ -49,20 +81,43 @@ func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
 		return err
 	}
 
-	res, resErr := c.do(
-		http.MethodPost,
+	res, err := c.do(
+		http.MethodPut,
 		fmt.Sprintf(dependenciesPathPattern, groupID, appID),
 		api.RequestOptions{
 			Body:        body,
 			ContentType: w.FormDataContentType(),
 		},
 	)
-	if resErr != nil {
-		return resErr
+	if err != nil {
+		return err
 	}
 	if res.StatusCode != http.StatusNoContent {
-		return api.ErrUnexpectedStatusCode{"import dependencies", res.StatusCode}
+		return fmt.Errorf("import got invalid status code %d", res.StatusCode)
```

Comment:
I could split this into a different command-based function like `deployDraftAndWait(...)` (e.g. `importDependenciesAndWait` but just having this in the function itself allows to be a drop-in replacement.

Reply:
for example, `DeployDraft` doesn't by itself wait for the draft to be deployed but `deployDraftAndWait` does. In this case `ImportDependencies` does always wait for the import to end. I was asking whether it'd be meaningful to have this separate as well. 
With regards to the error, my bad, I must have thought the api object was not of type `error`.

Reply:
Ah got it. I don't have a preference here. The way you have it now works for me.